<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SiSettingToolWid</class>
 <widget class="QWidget" name="SiSettingToolWid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="serialLab">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>COM0  未打开</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="serialBtn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>75</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>串口设置</string>
   </property>
  </widget>
  <widget class="QLabel" name="devLineLab">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>54</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>单相</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>54</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>相数：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="lineBtn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>75</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>级联数：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="devNumLab">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>240</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="devNumBtn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>270</y>
     <width>75</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdBtn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>370</y>
     <width>75</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>命令模式：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="devCmdLab">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>340</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>单命令</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
